Backend/lib/shared-services/package.yaml

```diff
@@ -112,6 +112,7 @@ dependencies:
   - beam-mysql
   - regex-compat
   - regex-tdfa
+  - external
```

Comment:
Why is this new dependency required ?

Reply:
that AWS flow  file is not in shared services for importing that file we need this dependency here .